snmp/datadog_checks/snmp/data/default_profiles/dell-os10.yaml

```diff
@@ -0,0 +1,231 @@
+extends:
+  - _base.yaml
+  - _generic-host-resources-base.yaml
+  - _generic-ucd.yaml
+#  - dell.yaml  # TODO
+#  - if32-mib.yaml  # TODO: later (https://datadoghq.atlassian.net/browse/NDM-1933)
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "dell"
```

Comment:
Similar to other cases like `_apc.yaml`, we can maybe move 

```
metadata:
  device:
    fields:
      vendor:
        value: "dell"
```

to _dell.yaml

And then, import `_dell.yaml` to all dell profiles e.g. dell.yaml, dell-os10.yaml, etc

Reply:
Yep, will do that once I figure out the hierarchy and extends :)